CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Crossover: use all attributes as hash inputs [#69](https://github.com/dusk-network/phoenix-core/issues/69)
+- Message: use all attributes as hash inputs [#69](https://github.com/dusk-network/phoenix-core/issues/69)
+- Update `canonical` from `v0.5` to `v0.6` [#72](https://github.com/dusk-network/phoenix-core/issues/72)
+- Change note position to reference [#76](https://github.com/dusk-network/phoenix-core/issues/76)
+
```

Comment:
It looks like you didn't address my comments in the previous review. This is not a valid format for either the changelog and the commit messages, please change it. This format is reserved to workspaces. Here it looks like `Message` and `Crossover` are members of a phoenix-core workspace.
Also, do not start with `use`.

Those should be rephrase into ```Change `Crossover` to use all attributes as hash inputs```, etc.

Modify the commit messages fa97e741c7a38a3050de929eaf57345c5f75664e and fa97e741c7a38a3050de929eaf57345c5f75664e accordingly.

Reply:
Fixed